<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DTMFMagix</class>
 <widget class="QMainWindow" name="DTMFMagix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>368</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>368</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>368</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DTMFMagix</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>300</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>361</width>
      <height>0</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="sendTab">
     <attribute name="title">
      <string>Send File</string>
     </attribute>
     <widget class="QPushButton" name="sendButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>161</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>161</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose File</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pathEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="pathLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>File path:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="requestTab">
     <attribute name="title">
      <string>Request File</string>
     </attribute>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>181</height>
       </rect>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="requestButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>101</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Request tree</string>
      </property>
     </widget>
     <widget class="QPushButton" name="downloadButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>101</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>331</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Progress:</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
